frontends/web/src/routes/account/receive/components/address-card.module.css

```
@@ -0,0 +1,48 @@
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  min-width: 0;
+}
+
+.address {
+  margin: 0;
+  font-family: var(--monospace, monospace);
+  font-size: 14px;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)